src/dmap_import/pipeline.py

```diff
@@ -113,6 +148,8 @@ def main() -> None:
         validate_db=True,
     )
 
+    check_for_parallel_tasks()
+
```

Comment:
Need a way to have this not run when testing locally. `running_in_aws` from `util_rds.py` could help